src/axom/primal/operators/closest_point.hpp

```diff
@@ -225,6 +226,27 @@ inline Point<T, NDIMS> closest_point(const Point<T, NDIMS>& pt,
   return Point<T, NDIMS>(res.array());
 }
 
+/*!
+ * \brief Computes the closest point from a point, P, to a sphere.
+ *
+ * \param [in] P the query point
+ * \param [in] sphere user-supplied triangle
+ * \return cp the closest point on sphere \a sphere to point P
+ *
+ * \note The closest point is uniquely defined everyone except at the sphere's origin.
+ * We handle that case by returning the point on the sphere along the direction
+ * determined by \a primal::Vector::unitVector() for the zero vector
```

Comment:
"for the unit vector" doesn't make sense to me

Reply:
Agreed. I'll clarify it.
The point is that at the center of a sphere, there is no unique closest point, so we choose an arbitrary one. `Vector::unitVector()` already handles this by using the vector that is 1 along the x-coordinate and 0 everywhere else.